MANUAL.txt

```diff
@@ -2465,7 +2465,7 @@ Currently the following pipes are predefined:
   left and right border.  Has no effect on other values. This
   can be used to align material in tables.  Widths are positive
   integers indicating the number of characters.  Borders
-  are strings inside double quotes; literal `"` and `\` characters
+  are strings inside double quotes; literal `"` and  characters
```

Comment:
Why this (undesirable) change?

Reply:
Whoops, that was unintentional - Pushing an update now :)